3p/integration.js

```diff
@@ -39,6 +39,7 @@ import {
 } from './3p';
 import {urls} from '../src/config';
 import {endsWith} from '../src/string';
+import {parseJson} from '../src/json';
```

Comment:
PR description calls this `jsonParse` while the import is called `parseJson`. I wonder if it's possible (or makes sense) to rename this to `jsonParse`, since it's more likely to show up in autocomplete when someone tries to type `JSON.parse` in an IDE.